### ISSUE TITLE
Debian package Reintroducing ?

### ISSUE THREAD

**Issue by javyer** [graygnuorg/pound#30]
Hello,

Thanks for the renewal of this nice proxy. Is there any intent to reintroduce in Debian Packaging, 
It was removed from testing in 2022:

```
=========================================================================
[Date: Mon, 10 Oct 2022 17:24:18 -0000] [ftpmaster: Thorsten Alteholz]
Removed the following packages from unstable:

     pound |    3.0.2-1 | source
     pound | 3.0.2-1+b1 | amd64, arm64, armel, armhf, i386, mips64el, mipsel, ppc64el, s390x
Closed bugs: 1020506

------------------- Reason -------------------
ROM; No further development and support from upstream after the end of the year, should not be part of the next Debian stable release
----------------------------------------------
```

Best Regards

**Comment by graygnuorg** [comment]
That doesn't depend on me, as a matter of fact.  I'm not sure what the proper procedure would be, but I have sent a mail to the recent debian maintainer asking him if he would be interested in reviving the package.

**Comment by graygnuorg** [comment]
Carsten (maintainer of the debian package) informed that he had uploaded the package to debian. It is not clear if it will be treated as a new package. If so it will take some time (days to weeks) to enter the "unstable" distribution.

**Comment by johndoe7000** [comment]
Carsten released pound 4.12 for Debian Sid some days ago.
This issue can be closed.

**Comment by javyer** [comment]
Thanks !!!
